cmd/restic/cmd_check.go

```diff
@@ -42,10 +46,27 @@ func init() {
 
 	f := cmdCheck.Flags()
 	f.BoolVar(&checkOptions.ReadData, "read-data", false, "read all data blobs")
+	f.UintSliceVar(&checkOptions.ReadDataSubset, "read-data-subset", nil, "read subset of data packs")
```

Comment:
Hm, maybe make it a string, and split it ourselves? The `*SliceVar` functions of `pflag` have some magic that bit us before, e.g. `--read-data-subset 23 --read-data-subset 24` is the same as `--read-data-subset 23,24`.

Maybe use `/` as the  delimiter? `1/5` reads (at least to me) as `one of five`)?